.github/workflows/build-then-deploy-docker.yml

```diff
@@ -0,0 +1,65 @@
+# Copyright 2023 Democratized Data Foundation
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.txt.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0, included in the file
+# licenses/APL.txt.
+
+# This workflow builds a Docker container image, if the build is successful
+# then it will deploy the image to DockerHub & GitHub container registries.
+name: Build Then Deploy Docker Workflow
+
+on: 
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+    branches:
+      - master
+      - develop
```

Comment:
question: Do we actually want a new image pushed to docker for every commit on develop? and on master?

todo: IMO pushing only for every tag might be sufficient, or if we do want docker container for the latest commit we can have a different action with that capability to build at the head commit on develop when we trigger that action manually.

```suggestion
```

Reply:
That's a good point. If we need dev images we can also manually build. I applied your change.